### PR TITLE
How to send and receive data from a RESTful API using a terminal and PHP

### PR DESCRIPTION
**Pull request by El-Karollo**
Hi. I just finished my second tutorial - about RESTful API. I look forward to checking out the article!

I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Karol Sarbiewski <1129032p10@gmail.com>

### CODE REVIEW

**Comment by El-Karollo** [comment]
UPS! Line 164 should be:
Now we need to tell cURL what type of request we want to send the data. In our case, it will be POST. That is, we add to the code:

My bad, sorry ^^

**Comment by svenja11** [comment]
Thank you for your contribution @El-Karollo! I read through your tutorial and I made some changes.

Please let me know if the changes I made are fine with you. If they are, please add the license block at the bottom of the file (`01.en.md`) and I will mark this PR as ready:

Remember to replace `[submitter's name and email address here]` with your own information.

```
##### License: MIT

<!--

Contributor's Certificate of Origin

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have
    the right to submit it under the license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my
    knowledge, is covered under an appropriate license and I have the
    right under that license to submit that work with modifications,
    whether created in whole or in part by me, under the same license
    (unless I am permitted to submit under a different license), as
    indicated in the file; or

(c) The contribution was provided directly to me by some other person
    who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are
    public and that a record of the contribution (including all personal
    information I submit with it, including my sign-off) is maintained
    indefinitely and may be redistributed consistent with this project
    or the license(s) involved.

Signed-off-by: [submitter's name and email address here]

-->
```

File: tutorials/communication-with-RESTful-API-from-terminal-and-PHP/01.en.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+</details>
+
+In this example, we saved the PHP code in a file called `example.php`:
+
+> Make sure you have the PHP curl extension installed (`php-curl`).
+
+```bash
+chmod +x example.php
+php example.php
```
Comment (wpdevelopment11) [inline tutorials/communication-with-RESTful-API-from-terminal-and-PHP/01.en.md:421]: I think that setting execute permission is only useful if you want to [use shebang](https://www.php.net/manual/en/features.commandline.usage.php#example-462).

```suggestion
php example.php
```

**Comment by El-Karollo** [comment]
Ive just edited tutorial. And yes, we dont need to give +x to php file.

and edits are good. I accept everything ;D

**Pull request merged**